Documentation/Books/Manual/ReleaseNotes/KnownIssues33.md

```diff
@@ -4,14 +4,12 @@ Known Issues in ArangoDB 3.3
-| 3 | **Date Added:** 2018-11-16 <br> **Component:** Security <br> **Deployment Mode:** All <br> **Description:** Unauthorized access to ArangoDB when using LDAP authentication <br> **Affected Versions:** 3.3.0 to 3.3.18 <br> **Fixed in Versions:** 3.3.19 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/223903752)  |
-| 2 | **Date Added:** 2018-11-03 <br> **Component:** Backup/Restore <br> **Deployment Mode:** All <br> **Description:** Users not included in the backup if _--server.authentication = true_ <br> **Affected Versions:** 3.3.0 to 3.3.13 <br> **Fixed in Versions:** 3.3.14 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/226557953) |
-| 1 | **Date Added:** 2018-04-09 <br> **Component:** Storage <br> **Deployment Mode:** Single Instance <br> **Description:** Data corruption could happen under Linux <br> **Affected Versions:** 3.3.0 <br> **Fixed in Versions:** 3.3.1 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/164069377)  |
+| 4 | **Date Added:** 2018-11-30 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Wrong suggestion printed in the log on how to optimize an OS setting, if followed, could cause ArangoDB to run into problems as the number of memory mappings will keep growing <br> **Affected Versions:** 3.3.0 to 3.3.19 <br> **Fixed in Versions:** 3.3.20 <br> **Reference:** [https://www.arangodb.com/alerts/tech03/](https://www.arangodb.com/alerts/tech03/)  |
+| 3 | **Date Added:** 2018-11-16 <br> **Component:** Backup/Restore <br> **Deployment Mode:** All <br> **Description:** Users not included in the backup if _--server.authentication = true_ <br> **Affected Versions:** 3.3.0 to 3.3.13 <br> **Fixed in Versions:** 3.3.14 <br> **Reference:** [https://www.arangodb.com/alerts/tech02/](https://www.arangodb.com/alerts/tech02/) |
+| 2 | **Date Added:** 2018-11-03 <br> **Component:** Security <br> **Deployment Mode:** All <br> **Description:** Unauthorized access to ArangoDB when using LDAP authentication <br> **Affected Versions:** 3.3.0 to 3.3.18 <br> **Fixed in Versions:** 3.3.19 <br> **Reference:** [https://www.arangodb.com/alerts/sec01/](https://www.arangodb.com/alerts/sec01/)  |
+| 1 | **Date Added:** 2018-04-09 <br> **Component:** Storage <br> **Deployment Mode:** Single Instance <br> **Description:** Data corruption could happen under Linux <br> **Affected Versions:** 3.3.0 <br> **Fixed in Versions:** 3.3.1 <br> **Reference:** [https://www.arangodb.com/alerts/tech01/](https://www.arangodb.com/alerts/tech01/)  |
```

Comment:
There does not seem to be any correlation between the number in the table in the number in the URL (e.g. `4` and `tech03`). Should we rather change this from a number to `tech03`?

Reply:
yeah, i also noticed - we can remove the number from this table in case, or how else we can correlate, the doc list is longer

Reply:
Oh, so there are issues in the docs for which there is no entry on the alert site? Not sure what to do then... But if the numbers have no real meaning, then we can also leave them out. The only benefit they provide is that - in communication - you can easily to refer to a specific issue without explaining what issue you mean. But then they need to be used consistently everywhere, otherwise they lose their meaning again.

Reply:
we agreed that we remove the numbers to reduce possible confusion. resolving